src/styles/header.scss

```diff
@@ -0,0 +1,60 @@
+.header {
+  width: 100%;
+  padding: 0 50px;
+  display: flex;
```

Comment:
It's recommended to use CSS Grid layout for the header instead of Flexbox. It provides more control over the alignment and distribution of elements.